<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangePwd</class>
 <widget class="QWidget" name="ChangePwd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>隶书</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>用 户 名  :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LineUserName">
        <property name="font">
         <font>
          <family>隶书</family>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>隶书</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>原 密 码  :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LineOriginalPwd">
        <property name="font">
         <font>
          <family>隶书</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>隶书</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>新 密 码  :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LineNewPwd">
        <property name="font">
         <font>
          <family>隶书</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_4" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>隶书</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>确认新密码:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LineSurePwd">
        <property name="font">
         <font>
          <family>隶书</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_5" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="SureBtn">
        <property name="text">
         <string>确认修改</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>325</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="CancelBtn">
        <property name="text">
         <string>返回</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
